junipervpn/vpn.py

```diff
@@ -25,8 +25,6 @@
 
 from junipervpn import tncc
 
-debug = False
```

Comment:
Should we make the equivalent change in `tncc.py` too?

And again, maybe this could be split out?